src/prommis/membrane_examples/nf/tests/test_nf_brine.py

```diff
@@ -0,0 +1,47 @@
+#################################################################################
+# adapted from test_nf.py
+#################################################################################
+
+"""
+Tests the nanofiltration flowsheet
+"""
+
+from pyomo.environ import value
+
+import pytest
+
+pytest.importorskip("watertap", reason="WaterTAP dependency not available")
+from prommis.membrane_examples.nf.nf_brine import main
```

Comment:
Apologies again, sometimes tools start arguing with each other and developers are caught in the middle... 

You can try using Pylint's "magic comment" directives to suppress this instance of the `wrong-import-position` message (will need to run Black to fix the spacing):

```suggestion
pytest.importorskip("watertap", reason="WaterTAP dependency not available")
# pylint: disable-next=wrong-import-position
from prommis.membrane_examples.nf.nf_brine import main
```

Reply:
Thank you for your help, I'll try this next. Do I need the `pytest.importorskip()` in my flowsheet scripts that call WaterTAP unit models? Or just in the test scripts?

Reply:
Just in the text scripts (i.e. files named `test_*.py`) should be enough.